src/components/App/index.tsx

```diff
@@ -2,12 +2,19 @@ import React from "react";
 import "./styles.scss";
 import LandingPage from "../Layouts/LandingPage";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { Quiz } from "../Quiz/index"; // File location finds the file "index" automatically
 
 const App: React.FC = () => {
   return (
     <Router>
+      {" "}
```

Comment:
I know the comments are useful to you but this starts to clog up the files for everyone, would you mind please removing them in all files? thank you